grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitCleanJobRepositoryServlet.groovy

```diff
@@ -0,0 +1,27 @@
+package com.twcable.grabbit.client.servlets
```

Comment:
Thinking out loud:

This servlet is temporary no?

What do you think about incorporating the DELETE verb into the grabbitJobServlet? That might be a more robust way to express this behavior. In addition, it is open to extension to the possibility of stopping running jobs